stats/src/main/java/io/airlift/stats/JmxGcMonitor.java

```diff
@@ -74,11 +81,24 @@ public void start()
         for (GarbageCollectorMXBean mbean : ManagementFactory.getGarbageCollectorMXBeans()) {
             ObjectName objectName = mbean.getObjectName();
             try {
-                ManagementFactory.getPlatformMBeanServer().addNotificationListener(
-                        objectName,
-                        notificationListener,
-                        null,
-                        null);
+                if (mbean.getName().startsWith("ZGC")) {
+                    // Don't bother registering ZGC Pauses because it does not report memory usage statistics
+                    if (!mbean.getName().endsWith("Pauses")) {
+                        zgcSupportsMajor |= mbean.getName().endsWith("ZGC Major");
+                        ManagementFactory.getPlatformMBeanServer().addNotificationListener(
```

Comment:
Static import `getPlatformMBeanServer()` and remove the wrapping

Reply:
done